scss/_utilities.scss

```
@@ -570,9 +575,12 @@ $utilities: map-merge(
       values: map-merge(
         $utilities-text-colors,
         (
-          "muted": $text-muted,
+          "muted": $text-muted, // deprecated
           "black-50": rgba($black, .5), // deprecated
           "white-50": rgba($white, .5), // deprecated
+          "body-secondary": var(--#{$prefix}secondary-color),
```

I need to double check but since we got this issue in Boosted with a similar case, I suppose that is might be the case here too.

```suggestion
          "body-secondary": rgba(var(--#{$prefix}secondary-color-rgb), var(--#{$prefix}text-opacity)),
```

Otherwise, using it with text opacity like in /utilities/colors/#example won't work for those 3 new values.

Note: it doesn't work already in the _main_ branch for `.text-muted`.